ballet/project.py

```diff
@@ -207,6 +207,12 @@ def branch(self):
     def on_master(self):
         return self.branch == 'master'
 
+    def on_master_after_merge(self):
+        if not self.on_master():
```

Comment:
can simply due `return self.on_master() and len(self.repo.active_branch.commit.parents) > 1`

Reply:
thanks!